docs/api/api_blueprint.apib

```diff
@@ -5251,3 +5250,36 @@ To update or get any attribute of this data layer, you will need admin access. H
             "self": "/v1/settings?id=1"
           }
         }
+
+# Group Upload
+
+This group of APIs contain endpoints for uploading media files. The workflow is different for images and any other files. In image, a temporary cropped image is first uploaded using the `Image Upload` endpoint. For other files, there can be a single file or multiple files for uploading.
+
+## Image Upload [/v1/upload/image{?force_local}]
+
+A temporary cropped image is first uploaded using this endpoint. After saving the image in it's prefered storage (local, google storage, s3, etc.), the URL for the temporary image is returned as response. This URL is then sent as form data to the necessary API, where it is then downloaded and resized and stored according to the need. *`force_local` parameter can be set to __true__ to force storage in local.* 
+
+
+### Upload an Image in temporary location [POST]
+
++ Parameters
+    + force_local (optional, string, `true`) - Force storage of temporary image in local
+
++ Request
+
+    + Headers
+
+            Content-Type: application/json
```

Comment:
No. Both the response and request are in the `application/json` format and not in `application/vnd.api+json` format since if you check my PR you will find that this URL is implemented separately from that of the rest jsonapi format.